crates/astria-core/src/primitive/v1/asset/denom.rs

```diff
@@ -691,4 +709,18 @@ mod tests {
 
         assert_eq!(None, denom.pop_leading_port_and_channel());
     }
+
+    #[test]
+    fn display_len_ok() {
```

Comment:
Yeah, thats what I had in mind. But in addition provide more exhaustive tests for some other scenarios. Use `#[track_caller]` to define the actual assertion and call it repeatedly from a test, similar to what's done here: https://github.com/astriaorg/astria/blob/f11f9000a28a4a2959ee488bfad73b751a4194c9/crates/astria-core/src/primitive/v1/mod.rs#L771-L796

Here is some background reading on this: https://matklad.github.io/2021/05/31/how-to-test.html#Test-Driven-Design-Ossification

Reply:
Thanks for the feedback and the link for background reading! I did this for both the core tests as well as the bridge lock tests, and attempted to be pretty exhaustive, but let me know if there are other cases that should be tested as well :)